modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -119,25 +119,22 @@ You can define scopes in three different places:
 ...
 ----
 
-Full https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset available on Exchange.
+For more information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
```

Comment:
Changes the meaning. There is not more info there, but the asset itself.

Reply:
Fixed!